### ISSUE TITLE
Improve non-interactive debugging

### ISSUE THREAD

**Issue by tysmith** [rr-debugger/rr#3748]
Most (all?) of the rr issues I report are found while running rr in non-interactive mode. These issues occur in automation and locally when using our tools that automate running Firefox and rr. Trying to reproduce these issues without using these tools can be difficult (sometimes impossible?) and time consuming.

To help both reporting and debugging these issues it would great if there was a way to dump helpful/required debug data to a log file while in non-interactive mode.

Perhaps a log file which is populated when there are fatal errors/assertions? Providing the file path via a command ling flag would be helpful for automation. Note using stderr is a pain when running in automation because the logs get mixed with the application that is running,

What do you think?

**Comment by rocallahan** [comment]
Yeah, more of this would be useful for CI too. I'll think about it.
